### PR TITLE
Added Sidhe fieldspawns, Coyote Ai and fixed KoboldBanditAi

### PR DESCRIPTION
**Pull request by Sairii**
Coyote AI is for all coyote, dingo and jackal types.
It is a "jumpier" or a more "agile" wolf ai.

### CODE REVIEW

File: system/scripts/ai/coyote.cs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+		}
+		else if (rndAlert < 45) // 5%
+		{
+			Do(Attack(3, 4000));
+		}
+		else if (rndAlert < 90) // 45%
+		{
+			if (Random() < 50)
+			{
+				Do(Circle(500, 1000, 4000, true));
```
Comment (exectails) [inline system/scripts/ai/coyote.cs]: I just fixed those in your two other AIs... what you're doing here is make them either walk or run in circles, don't blindly copy official AIs. The next one is technically correct, but it's pointless, if you use this version the 50% clockwise thing is done behind the scenes.

File: system/scripts/ai/koboldbandit.cs [hunk 2]
```diff
@@ -164,13 +164,27 @@ protected override IEnumerable Alert()
 		{
 			if (Random() < 80)
 				Do(Say(koboldBanditAlert));
-			Do(Circle(500, 1000, 4000, false));
```
Comment (exectails) [inline system/scripts/ai/koboldbandit.cs]: You can't be serious...

Comment (Sairii) [inline system/scripts/ai/koboldbandit.cs]: o.O

**Comment by Sairii** [comment]
test

File: system/scripts/regions/sidhe.cs [hunk 3]
```diff
@@ -15,16 +15,24 @@ public override void LoadProperties()
...
-		// ...
+		// Southern Fields
+		CreateSpawner(race: 20301, amount: 7, region: 47, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(7535, 7296, 7535, 11284, 12529, 11284, 12529, 7296)); // Coyote
+		CreateSpawner(race: 20301, amount: 7, region: 47, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(7151, 14799, 7151, 17485, 12240, 17485, 12240, 14799)); // Coyote
+		CreateSpawner(race: 20301, amount: 5, region: 47, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(6379, 11752, 6379, 15262, 13352, 15262, 13352, 11752)); // Coyote
+
+		// Northern Fields
+		CreateSpawner(race: 20301, amount: 7, region: 48, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(8714, 15997, 12860, 8391, 11789, 7807, 7642, 15413)); // Coyote
+		CreateSpawner(race: 20301, amount: 5, region: 48, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(7498, 15867, 7498, 23639, 8938, 23639, 8938, 15867)); // Coyote
+		//CreateSpawner(race: 20301, amount: 5, region: 48, delay: 100, delayMin: 10, delayMax: 20, coordinates: A(7274,23943, 9210,28627, 10612,28047, 8675,23363)); // Coyote disabled on official
```
Comment (exectails) [inline system/scripts/regions/sidhe.cs]: What's wrong about this last spawner?

**Pull request merged**